src/__tests__/__views__/__snapshots__/SignUp.test.js.snap

```diff
@@ -0,0 +1,256 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This file should be put in .gitignore with other snapshots' test files because it is generated every time you run your tests.

Reply:
The file is not generated every time I run my tests
it's generated only when I run the test with the update argument. Like `yarn test -u`!
It is better to keep it under git so that everyone can be aware when the component is changed.

Reply:
@espoirMur The snapshot is the representation of the component. Component changes appear both in the snapshot and the component. Let us keep track of the component's changes from the component itself to avoid changes duplication and keep our code base clean.

Reply:
I have fixed the this while rebasing.